usaspending_api/api_contracts/contracts/v2/agency/toptier_code/obligations_by_type.md

```diff
@@ -0,0 +1,58 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Federal Obligations [/api/v2/agency/{toptier_code}/obligations_by_type/{?fiscal_year}]
+
+This endpoint is used to power USAspending.gov's agency profile pages.
+
+## GET
+
+This endpoint returns a breakdown of obligations by award type (contracts, IDVs, grants, loans, direct payments, other) within a fiscal year.
+
++ Parameters
+    + `toptier_code`: 086 (required, number)
+        The toptier code of an agency (could be a CGAC or FREC) so only numeric character strings of length 3-4 are accepted.
+    + `fiscal_year`: 2017 (required, number)
+        The fiscal year that you are querying data for.
+        
++ Response 200 (application/json)
+    + Attributes (object)
+        + `award_obligations` (required, number)
+        + `results` (required, array[ObligationSubtotals], fixed-type)
+
+    + Body
+
+            {
+                "award_obligations": 39999999.96,
+                "results": [
+                    {
+                        "type": "contracts",
+                        "obligated_amount": 9999999.99
```

Comment:
A few similar endpoints to this one return the monetary values as `aggregated_amount`

Reply:
@brett-varney did you want to keep the response as-is or was this comment overlooked?

Reply:
@tony-sappe I already changed them...?

Reply:
Contract was changed to `award_obligations`

Reply:
@tony-sappe OK, not really sure where I got that... fixed